pom.xml

```diff
@@ -187,12 +193,24 @@
           <excludeFilterFile>${findbugs.config}</excludeFilterFile>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>com.github.spotbugs</groupId>
+        <artifactId>spotbugs-maven-plugin</artifactId>
+        <version>${spotbugs.maven.plugin.version}</version>
+        <configuration>
+          <encoding>${project.build.sourceEncoding}</encoding>
+          <xmlOutput>true</xmlOutput>
+          <effort>Max</effort>
+          <threshold>Low</threshold>
+          <excludeFilterFile>${spotbugs.config}</excludeFilterFile>
+        </configuration>
+      </plugin>
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-checkstyle-plugin</artifactId>
         <version>${maven.checkstyle.plugin.version}</version>
         <configuration>
-          <configLocation>file://${project.build.directory}/plugin-config/checkstyle/hotels-checkstyle.xml</configLocation>
+          <configLocation>${checkstyle.config}</configLocation>
```

Comment:
OK, I see why you used this above. I guess one can do either, I think the shortened version I used above requires less config and is how we did it in our internal parent pom which seems to work.